src/EntityComponentManager.cc

```diff
@@ -1236,8 +1236,22 @@ void EntityComponentManager::SetState(
       {
         std::istringstream istr(compMsg.component());
         comp->Deserialize(istr);
-        this->SetChanged(entity, compIter.first,
-            ComponentState::OneTimeChange);
+        // \todo(anyone) Note on merging forward:
```

Comment:
I don't know if this is a todo, since this is already the case in `ign-gazebo4` forward.  Maybe just a note is sufficient?

Reply:
yep done. 03446c6